### PR TITLE
Add tampa file

### PR DESCRIPTION
**Pull request by githubteacher**
Adding the file that highlights places to go in Tampa / Tarpon Springs.

Closes https://github.com/githubteacher/sep-22-developers/issues/4 
Resolves https://github.com/githubteacher/sep-19-developers/issues/55

### CODE REVIEW

**Comment by githubteacher** [comment]
@hectorsector What do you think of my contribution? Do you need any more info?

File: githubteacher-tampa.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Tampa, Fl
+
+### Places Visit
+
+
+
+### Places to Eat
+- Dunedin Brewery for :tacos: & :beer:
+- The Place for sandwiches
+- Demitiri's on the Water
```
Comment (hectorsector) [inline githubteacher-tampa.md:10]: Is this the place for Sponges, @githubteacher?

**Comment by hectorsector** [comment]
Also, not enough information about tacos.

File: githubteacher-tampa.md (hunk 1 above)
Comment (githubteacher) [reply githubteacher-tampa.md:10]: No, that is a restaurant, but I will add the Sponge Docks as a place to visit. 👍

Comment (hectorsector) [reply githubteacher-tampa.md:10]: Ah, so no eating sponges then 😉

**Comment by githubteacher** [comment]
@hectorsector Made some changes, is this good 🚢:shipit:  ? Just need a 👍

**Comment by hectorsector** [comment]
👍

**Pull request merged**